src/client/eppo-client.ts

```diff
@@ -103,6 +103,7 @@ export type EppoClientParameters = {
   flagConfigurationStore: IConfigurationStore<Flag | ObfuscatedFlag>;
   banditVariationConfigurationStore?: IConfigurationStore<BanditVariation[]>;
   banditModelConfigurationStore?: IConfigurationStore<BanditParameters>;
+  overrideStore?: ISyncStore<Variation>;
```

Comment:
Is this intended change? seems unrelated to obfuscation

Reply:
It is unrelated, but I thought I could squeeze this fix in here. I added it to the PR. I am ok with opening a new PR if you disagree with piggybacking here.